XenAdmin/Wizards/NewSRWizard_Pages/ChooseSrProvisioningPage.cs

```diff
@@ -66,13 +67,26 @@ public bool IsGfs2
 
         public override void PopulatePage()
         {
-            var gfs2Allowed = !Helpers.FeatureForbidden(Connection, Host.RestrictGfs2) && Connection.Cache.Clusters.Any();
+            var clusteringEnabled = Connection.Cache.Clusters.Any();
+            var gfs2Allowed = !Helpers.FeatureForbidden(Connection, Host.RestrictGfs2) && clusteringEnabled;
 
             radioButtonGfs2.Enabled = labelGFS2.Enabled = gfs2Allowed;
             tableLayoutInfo.Visible = radioButtonLvm.Checked = !gfs2Allowed;
             labelWarning.Text = Helpers.FeatureForbidden(Connection, Host.RestrictGfs2)
                 ? Messages.GFS2_INCORRECT_POOL_LICENSE
                 : Messages.GFS2_REQUIRES_CLUSTERING_ENABLED;
+            linkLabelPoolProperties.Visible = !clusteringEnabled && !Helpers.FeatureForbidden(Connection, Host.RestrictGfs2);
+        }
+
+        private void linkLabelPoolProperties_LinkClicked(object sender, System.Windows.Forms.LinkLabelLinkClickedEventArgs e)
+        {
+            var pool = Helpers.GetPoolOfOne(Connection);
+
+            using (PropertiesDialog propertiesDialog = new PropertiesDialog(pool))
+            {
+                propertiesDialog.SelectClusteringEditPage();
+                propertiesDialog.ShowDialog(this);

```

Comment:
When using this link to enable clustering, after the pool properties is closed and clustering is enabled, we need to refresh this page, so that thib provisioning is allowed and the warning is hidden.